### PR TITLE
add TODO

### PR DESCRIPTION
**Pull request by whitewum**
as in https://github.com/vesoft-inc/nebula-docs/pull/379/files

fix some todo

### CODE REVIEW

File: docs-2.0/1.introduction/1.what-is-nebula-graph.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ Nebula Graph采用shared-nothing架构，支持在不停止数据库服务的情
 
 ### 易开发
 
-Nebula Graph提供Java、Python、C++和Go等流行编程语言的客户端，更多客户端仍在开发中。详情请参见Nebula Graph clients（TODO: doc）。
+Nebula Graph提供Java、Python、C++和Go等流行编程语言的客户端，更多客户端仍在开发中。详情请参见[Nebula Graph clients](../2.quick-start/6.useful-links.md)。
 
 ### 高可靠访问控制
 
-Nebula Graph支持严格的角色访问控制和LDAP（Lightweight Directory Access Protocol）等外部认证服务，能够有效提高数据安全性。详情请参见验证和授权（TODO: doc）。
+Nebula Graph支持严格的角色访问控制和LDAP（Lightweight Directory Access Protocol）等外部认证服务，能够有效提高数据安全性。详情请参见[验证和授权 TODO](../7.data-security/1.authentication/1.authentication.md)。
```
Comment (randomJoe211) [inline docs-2.0/1.introduction/1.what-is-nebula-graph.md:35]: 这里还有TODO没删掉

File: docs-2.0/2.quick-start/1.quick-start-workflow.md [hunk 2]
```diff
@@ -11,5 +11,6 @@
 以下是其他推荐您阅读的文档。在您尝试过快速入门的操作后，您可能会需要阅读它们来进一步了解和使用Nebula Graph。
 
 - [常见问题FAQ](./0.FAQ.md)
-- 部署Nebula Graph集群 (TODO: doc）
+- [部署Nebula Graph集群 TODO](../4.deployment-and-installation/deploy-nebula-graph-cluster.md)
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/1.quick-start-workflow.md:14]: 这里也还有TODO

File: docs-2.0/2.quick-start/6.useful-links.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* [HTTP Client](https://github.com/vesoft-inc/nebula-http-gateway)
+
+## Graph tools
+
+| links | commit id |
+| -- | -- |
+| [Command Line Console](https://github.com/vesoft-inc/nebula-console/tree/v2.0.0-ga)  |  1f32236 |
+
+The follow repositories of v2.0.0 are not released yet. (TODO Mar. 23 2021):
+* [Studio](https://github.com/vesoft-inc/nebula-docker-compose)
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/6.useful-links.md:24]: This is the repo for nebula-docker-compose, not Studio.
Correct link: https://github.com/vesoft-inc/nebula-web-docker

File: docs-2.0/2.quick-start/6.useful-links.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+* [Dashboard](https://github.com/vesoft-inc/nebula-stats-exporter)
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/6.useful-links.md:25]: Dashboard is on https://github.com/vesoft-inc-private/nebula-dashboard
The one you have here is for Nebula Stats Exporter

File: docs-2.0/2.quick-start/6.useful-links.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+* [Graph Computing TODO]
+
+## Big Data and other Systems support
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/6.useful-links.md:28]: No algorithm？

File: docs-2.0/2.quick-start/6.useful-links.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+* [Node.js Client](https://github.com/vesoft-inc/nebula-node)
+* [HTTP Client](https://github.com/vesoft-inc/nebula-http-gateway)
+
+## Graph tools
+
+| links | commit id |
+| -- | -- |
+| [Command Line Console](https://github.com/vesoft-inc/nebula-console/tree/v2.0.0-ga)  |  1f32236 |
+
+The follow repositories of v2.0.0 are not released yet. (TODO Mar. 23 2021):
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/6.useful-links.md:23]: The following repositories. Please fix this in the whole topic.

File: docs-2.0/2.quick-start/6.useful-links.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+## Big Data and other Systems support
+
+| links | commit id |
+| -- | -- |
+| [csv (a.k.a. importer)](https://github.com/vesoft-inc/nebula-importer/tree/release-v2.0.0-ga) | 1d87c7b |
+| [Spark](https://github.com/vesoft-inc/nebula-spark-utils/tree/v2.0.0) | af3fdf4 |
+The follow repositories of v2.0.0 are not released yet. (TODO Mar. 23 2021):
+  
+* [Flink](https://github.com/vesoft-inc/nebula-flink-connector)
+* [Promethus](https://github.com/vesoft-inc/nebula-stats-exporter)
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/6.useful-links.md:37]: Nebula Stats Exporter or Monitor with Promethus?

File: docs-2.0/2.quick-start/6.useful-links.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+* [ElasticSearch](../4.deployment-and-installation/6.deploy-text-based-index/2.deploy-es.md)
+
+## Benchmark, test, and Backup tools
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/6.useful-links.md:40]: Why capitalize Backup?

**Review by randomJoe211 — CHANGES_REQUESTED** [review]

File: docs-2.0/1.introduction/1.what-is-nebula-graph.md (hunk 1 above)
Comment (whitewum) [reply docs-2.0/1.introduction/1.what-is-nebula-graph.md:35]: 因为这个中文还没写啊。。。

File: docs-2.0/2.quick-start/1.quick-start-workflow.md (hunk 2 above)
Comment (whitewum) [reply docs-2.0/2.quick-start/1.quick-start-workflow.md:14]: 因为这个中文还没写啊。。。

**Comment by whitewum** [comment]
right

File: docs-2.0/1.introduction/1.what-is-nebula-graph.md (hunk 1 above)
Comment (randomJoe211) [reply docs-2.0/1.introduction/1.what-is-nebula-graph.md:35]: 有的啊，就这个路径，现有中文的~

File: docs-2.0/2.quick-start/1.quick-start-workflow.md (hunk 2 above)
Comment (randomJoe211) [reply docs-2.0/2.quick-start/1.quick-start-workflow.md]: 也有了啊

File: docs-2.0/2.quick-start/6.useful-links.md (hunk 5 above)
Comment (whitewum) [reply docs-2.0/2.quick-start/6.useful-links.md:28]: no currently

File: docs-2.0/2.quick-start/6.useful-links.md (hunk 7 above)
Comment (whitewum) [reply docs-2.0/2.quick-start/6.useful-links.md:37]: It's OK

**Review by randomJoe211 — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**